src/main/java/org/fundacionjala/at15/katas/bankocr/vladimir/MostrarDatosImpl.java

```diff
@@ -0,0 +1,337 @@
+package org.fundacionjala.at15.katas.bankocr.vladimir;
+
+import java.util.Arrays;
+
+public class MostrarDatosImpl {
```

Comment:
The code in this class is very well ordered and it is not difficult to understand, however, it is getting too long. You should consider refactor this in order to broke it in smaller pieces (classes/methods)

Reply:
ok i will do it